index.d.ts

```diff
@@ -32,6 +42,7 @@ export type ResponseData = [number, { [k: string]: string }, string];
 interface ExtraRequestData {
   params: Params;
   queryParams: QueryParams;
+  requestBody: string
```

Comment:
https://github.com/pretenderjs/pretender/blob/v3.0.2/src/index.ts#L396
Pretender only adds `params` and `queryParams`. I think this needs to be fixed in `fake-xml-http-request` package.

Reply:
https://github.com/pretenderjs/FakeXMLHttpRequest/pull/48